nav2_regulated_pure_pursuit_controller/src/parameter_handler.cpp

```diff
@@ -85,6 +85,8 @@ ParameterHandler::ParameterHandler(
     node, plugin_name_ + ".rotate_to_heading_min_angle", rclcpp::ParameterValue(0.785));
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".max_angular_accel", rclcpp::ParameterValue(3.2));
+  declare_parameter_if_not_declared(
+    node, plugin_name_ + ".cancel_deceleration", rclcpp::ParameterValue(1.0));
```

Comment:
```suggestion
    node, plugin_name_ + ".cancel_deceleration", rclcpp::ParameterValue(3.2));
```

Also, this param needs to be added to the RPP configuration guide

Reply:
I've added this to the documentation PR